doc/api/fs.md

```diff
@@ -13,1160 +13,1284 @@ way modeled on standard POSIX functions.
 
 To use this module:
 
-```js
-const fs = require('fs');
+```js esm
+// Using ESM Module syntax:
+import * as fs from 'fs';  // Sync and callback APIs
```

Comment:
I'd prefer not to. But I'll leave that up to @nodejs/modules

Reply:
I would also prefer not to; specifiers being URLs is an obscure implementation detail.

Reply:
I wouldn't present it as a URL specifier, but rather as a prefix for built-in modules, to avoid name clashes with the ecosystem when we introduce new modules (refs: https://github.com/nodejs/node/issues/36098). I'm fine whatever we chose, I just wanted to throw the idea out there.

Reply:
If we’re going to start using `node:` in specifiers, that should perhaps be its own global PR to update all sections at once?